src/up.js

```diff
@@ -27,6 +28,13 @@ module.exports = function up() {
     description: 'Run a Helix development server',
     builder: (yargs) => {
       yargsBuild(yargs);
+      yargsParams(yargs, {
+        name: 'dev-default',
+        describe: 'Additional action parameters',
+        alias: ['action-params', 'actionParams', 'devDefault'],
```

Comment:
I think we don't need the `acton-param` and `actionParams` aliases anymore.

Reply:
Ok, I'll erase it :D